extensions/parquet/table/src/main/java/io/deephaven/parquet/table/layout/ParquetFileHelper.java

```diff
@@ -5,14 +5,21 @@
 
 import io.deephaven.parquet.table.ParquetTableWriter;
 
-import java.nio.file.Path;
+import java.net.URI;
+import java.util.regex.Pattern;
 
 final class ParquetFileHelper {
     /**
      * Used as a filter to select relevant parquet files while reading all files in a directory.
      */
-    static boolean fileNameMatches(final Path path) {
-        final String fileName = path.getFileName().toString();
-        return fileName.endsWith(ParquetTableWriter.PARQUET_FILE_EXTENSION) && fileName.charAt(0) != '.';
+    private static final Pattern HIDDEN_FILE_PATTERN = Pattern.compile("(^|/)\\.[^/]+");
+
+    static boolean isNonHiddenParquetURI(final URI uri) { // TODO better names for this file
```

Comment:
I'm not a fan of "not", "non", nor other inversion language in method names typically. I was confused at first.

Maybe a better name would be `isVisibleParquetURI`?